process/logsevents/fungibleESDTProcessor.go

```diff
@@ -39,6 +44,12 @@ func (fep *fungibleESDTProcessor) processEvent(args *argsProcessEvent) (string,
 	}
 
 	topics := args.event.GetTopics()
+	nonceBig := big.NewInt(0).SetBytes(topics[1])
+	if nonceBig.Uint64() > 0 {
+		// is a semi-fungible token should return
```

Comment:
`// this is a semi-fungible token so we should return` maybe

Reply:
done.